routes/index.js

```diff
@@ -255,6 +255,43 @@ module.exports = function () {
 
   // ===== /end docs
 
+  // Awesome
+  router.get(['/awesome', '/awesome/*'], (req, res) => {
+    const BASE = 'https://raw.githubusercontent.com/datahubio/awesome/master/';
+    let path = urllib.parse(req.path).path;
+    let fullpath;
+    if (path == '/awesome'){
+      fullpath = BASE + 'index.md'
+    } else {
+      // get the page name from the path and create a full adress
+      fullpath = BASE + path.split('/')[2] + '.md';
+    }
+    //request raw page from github
+    fetch(fullpath)
+        .then(function(res) {
+            return res.text();
+        })
+        .then(function(text) {
+            // parse the raw .md page and render it with an template.
+            const parsedWithFM = fm(text);
+            const content = utils.md.render(parsedWithFM.body);
+            const date = parsedWithFM.attributes.date
```

Comment:
We don't use the date and the frontmatter should not have a date. Structure of an awesome page should be:

```
title:  The Best Football Datasets
description: // goes into the meta tag (this is the for future and can be left out for now)
```